Building-ChakraCore.md

```diff
@@ -113,3 +113,17 @@ Some details on build script arguments used above:
 * ```--icu=..``` specify the location of ICU library (use `--no-icu` if you don't need unicode support)
 * ```--test-build``` enable command line options and some optimizations
 * ```-j=2``` compile with 2 threads
+
+### Building and debugging in XCode ###
+
+Generate an XCode project:
+* ```./build.sh --xcode --icu=/usr/local/opt/icu4c/include --debug --static```
```

Comment:
Would add an extra step of installing dependencies and cloning ChakraCore as instructed above just to make things clear.